app/javascript/packs/form-validation.ts

```diff
@@ -3,12 +3,14 @@ import { t } from '@18f/identity-i18n';
 /**
  * Given a submit event, disables all submit buttons within the target form.
  *
- * @param {Event} event Submit event.
+ * @param event Submit event.
  */
-function disableFormSubmit(event) {
-  const form = /** @type {HTMLFormElement} */ (event.target);
-  Array.from(form.querySelectorAll('button:not([type]),[type="submit"]')).forEach((submit) => {
-    /** @type {HTMLInputElement|HTMLButtonElement} */ (submit).disabled = true;
+function disableFormSubmit(event: Event) {
+  const form = event.target as HTMLFormElement;
+  Array.from(form.querySelectorAll('button:not([type]),[type="submit"]')).forEach((element) => {
```

Comment:
I see it's just picked and moved from existing and I'm unsure if we have any notion of a selector styleguide, but personally I'd add in a space here to make it easier to skim:
```suggestion
  Array.from(form.querySelectorAll('button:not([type]), [type="submit"]')).forEach((element) => {
```

Reply:
Maybe it's getting a bit carried away, but we could also build the selector from an array of strings? To further reinforce that it's a combination of individual selectors, and so that it's in the domain of Prettier to manage the spacing styling.

```suggestion
  Array.from(form.querySelectorAll(['button:not([type])', '[type="submit"]'].join()).forEach((element) => {
```

Reply:
I like that!

Reply:
Updated in f15bb4059.